src/app-data/index.ts

```diff
@@ -108,7 +108,7 @@ export const BUILD: BuildConditionals = {
   transformTagName: false,
   attachStyles: true,
   // TODO(STENCIL-914): remove this option when `experimentalSlotFixes` is the default behavior
-  patchPseudoShadowDom: false,
+  experimentalSlotFixes: false,
```

Comment:
Just decided to swap out this build flag with a more generic one to put all our vDOM/slot changes behind rather than adding new flags for each fix. All this happens in 3d7029a97991ab62a5bc8b79449fb24985825a83